pytest_fixtures/component/provision_pxe.py

```diff
@@ -11,6 +11,9 @@
 from robottelo import constants
 from robottelo.api.utils import enable_rhrepo_and_fetchid
 from robottelo.api.utils import wait_for_tasks
+from robottelo.cli.factory import make_product
+from robottelo.cli.factory import make_repository
```

Comment:
The use of `cli.factory` is prohibited unless required. 

Use this instead `sat_obj.cli_factory.make_product` and same for next!